utilities/rpp-unittests/HIP_NEW/BatchPD_hip_pkd3.cpp

```diff
@@ -2549,7 +2549,7 @@ int main(int argc, char **argv)
         Rpp32u kernelSize[images];
         for (i = 0; i < images; i++)
         {
-            kernelSize[i] = 5;
```

Comment:
do we have separate test cases for kernel_size=5?

Reply:
Yes we do, I am testing all sub-variants in kSize=3/5/7/9 for the new Tensor code.